raspberry/bzzz/sensors/gps_lea6s.py

```diff
@@ -89,10 +98,19 @@ def longitude(self):
         GPS data object
         """
         return self.__gpgll_longitude
+    
+    @property
+    def altitude(self):
+        """
+        GPS data object
+        """
+        return self.__gpgsv_altitude
 
-my_sensor = GpsLea6SReader(serial_path="/dev/tty.usbmodem142201")
+my_sensor = GpsLea6SReader(serial_path="/dev/tty.usb...")
```

Comment:
Call the sensor using a context - i.e., 

```python
with GpsLea6SReader(serial_path="/dev/tty.usb...") as mysensor:
   # your code goes here
```

Reply:
the wrong version of the code has been committed I will fix now

Reply:
very strange, it says everything has been committed on my branch, but it has not on git hub for some reason...

Reply:
all fixed now